File: C/section08/S08E01.c
#include <stdio.h>

int main() { 
    int vectors[5], evenAndAboveZero[5], evenCounter = 0;

    for (int counter = 0; counter < 5; counter++) {
    
        printf("Please enter a number: \n");
        scanf("%d", &vectors[counter]);

        if (vectors[counter] > 0 && vectors[counter] % 2 == 0) {
            evenAndAboveZero[evenCounter] = vectors[counter];
            evenCounter++;
        }

    }

    for (int counter = 0; counter < evenCounter; counter++) {
        
        printf("\n%d", evenAndAboveZero[counter]);
               
    }

    return 0;    
}

File: Python/section08/S08e01.py
vector = []
evenNumbers = []

for index in range(0,10):
    vector.append(int(input("Please enter a number {0}/10: ".format(index+1))))
    if vector[index] % 2 == 0:
        evenNumbers.append(vector[index])

print("\nList of even numbers")
for index in evenNumbers:
    print("Even number: {0}".format(index))